src/ExceptionHandler.php

```diff
@@ -297,9 +300,11 @@ public function report(Throwable $e): void
             }, $e->getErrors());
         }
 
+        $message = "{$e->getMessage()}, called in {$e->getFile()} on line {$e->getLine()}";
```

Comment:
Note: this is the classical way PHP transforms an exception to a string but without the trace. It is very handy to have that in the title because then you can navigate quickly to the place of exception.

Reply:
> There was a problem with the request to the carrier. The original response can be found in the meta under carrier_response.

I have seen this exact error pop up in the backoffice, I believe as shipment status errors (might be caused when a microservice returns a 500). We should avoid putting sensitive information about the code/path in the description.